Classes/Backend/DynamicFlexForm.php

```diff
@@ -162,6 +162,10 @@ public function parseDataStructureByIdentifierPreProcess(array $identifier)
         } else {
             $dataSourceIdentity = $identifier['tableName'] . '.' . $identifier['fieldName'] . ':' . $identifier['uid'];
         }
+        if(!isset(static::$generatedDataSources[$dataSourceIdentity])) {
```

Comment:
Thanks, i´ll leave the refactoring to you. I am a bit confused about the usage of the identifier array. Maybe this could become a small data storage class like the ``TemplatePaths`` .. ?